.buildkite/pipeline.cleanamis.yaml

```diff
@@ -0,0 +1,19 @@
+steps:
+  - name: ":broom: Delete AMIs ({{matrix}})"
+    command: .buildkite/steps/clean-old-amis
+    agents:
+      queue: "oss-deploy"
+    env:
+      DRY_RUN: true
```

Comment:
for now, we're only dry running it. We'll build confidence the output looks like we expect, then remove this